openfecwebapp/templates/partials/candidate/itemized-disbursements.html

```diff
@@ -0,0 +1,70 @@
+{% import 'macros/missing.html' as missing %}
+{% import 'macros/null.html' as null %}
+{% import 'macros/cycle-select.html' as select %}
+
+<section id="section-4" role="tabpanel" aria-hidden="true" aria-labelledby="section-4-heading">
+  <div class="heading--section">
+    <h2 id="section-4-heading">Itemized disbursements</h2>
+  </div>
+
+  <div class="content__section">
+    {{ select.candidate_cycle_select(cycles, cycle, id="cycle-4") }}
+  </div>
+
+  <span class="t-sans t-bold">Data is included from these committees:</span>
+
+  <ul class="list--bulleted">
+    {% for committee in committee_groups['P'] | reverse %}
+    <li>
+      <a class="t-sans" href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle) }}">{{ committee.name }}</a>
+    </li>
+    {% endfor %}
+    {% for committee in committee_groups['A'] | reverse %}
+    <li>
+      <a class="t-sans" href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle) }}">{{ committee.name }}</a>
+    </li>
+    {% endfor %}
+  </ul>
+
+  <div class="content__section">
+    <div class="results-info results-info--simple">
+      <h3 class="u-no-margin">Disbursements by transaction</h3>
+
+      <a class="u-float-right button--alt button--browse"
+          href="{{ url_for(
+            'individual_contributions',
+            two_year_transaction_period=cycle,
+            min_date=cycle_start(cycle) | date(fmt='%m-%d-%Y'),
+            max_date=cycle_end(cycle) | date(fmt='%m-%d-%Y')
+          ) }}{% for c in committee_groups['P'] | reverse %}&committee_id={{ c.committee_id }}{% endfor %}{% for c in committee_groups['A'] | reverse %}&committee_id={{ c.committee_id }}{% endfor %}">Filter this data</a>
+
+      <ul>
+        <li class="tag__category">
+          <div class="tag__item">Report year: {{cycle - 1}} to {{cycle}}</div>
+        </li>
+      </ul>
+
+      <button type="button" class="js-export button button--cta button--export" data-export-for="itemized-disbursements">Export</button>
+    </div>
+
+    <table
+        class="data-table data-table--heading-borders"
+        data-type="itemized-disbursements"
+        data-committee-id="{% for c in committee_groups['P'] | reverse %}{{ c.committee_id }},{% endfor %}{% for c in committee_groups['A'] | reverse %}{{ c.committee_id }},{% endfor %}"
```

Comment:
Just realized that there's already `committee_ids` coming from the view, so I just made a simpler loop that also works for the Filter this data button url as well.